frontend/packages/dev-console/src/components/topology/shapes/BaseNode.tsx

```diff
@@ -88,15 +90,16 @@ export default class BaseNode extends React.Component<BaseNodeProps, State> {
             }
           />
           {label != null && (
-            <text
+            <SvgBoxedText
               className="odc-base-node__label"
-              textAnchor="middle"
-              y={outerRadius + 10}
+              y={outerRadius + 20}
               x={0}
-              dy="0.6em"
+              paddingX={10}
+              paddingY={3}
+              icon={<ResourceIcon kind="BuildConfig" />}
```

Comment:
The icon looks to be part of a separate PR. If you remove all icon related changes, you can add the border now and then add the icon in a later PR.

This is also wrong because you've hard coded a kind.

Reply:
oops! this is not part of this PR. I was trying this and pushed it accidently. :P. Agreed this will be part of a seperate PR